README.md

```diff
@@ -38,9 +38,9 @@ In your Maven Java project, add the plugin to your `pom.xml`:
 
 ### Configuration
 
-Configure the plugin by changing `registry`, `repository`, and `credentialHelperName` accordingly.
+Configure the plugin by changing `registry`, `repository`, and `credentialHelperName` in the registry of your choice. For example:
```

Comment:
"in the registry of your choice" might not be the right wording here, since `repository` is not specific to a registry. Perhaps something like "...changing `registry`, `repository`, and `credentialHelperName`. (See [Extended Usage]() for details.)" might fit better?

Reply:
What if we said something like:
Configure the plugin by changing `registry`, `repository`, and `credentialHelperName` in your registry.